lib/ansible/plugins/strategy/__init__.py

```diff
@@ -541,6 +543,8 @@ def search_handler_blocks_by_name(handler_name, handler_blocks):
                                 for listening_handler_block in iterator._play.handlers:
                                     for listening_handler in listening_handler_block.block:
                                         listeners = getattr(listening_handler, 'listen', []) or []
+                                        if not listeners:
+                                            continue
                                         templar = self.get_handler_templar(listening_handler, iterator)
```

Comment:
Yeah, the templar isn't needed or used in this case so just reusing the dummy templar should be fine. `listen: "{{ var }}"` has never worked.
get_validated_value() just enforces that listeners is a list and contains string types.

Reply:
:+1: